src/components/Footer.tsx

```diff
@@ -158,7 +158,7 @@ const Footer: React.FC = () => {
       </div>
       <div className={styles.copyrightContent}>
         <div className={styles.copyright}>
-          <p>&copy; 2024 &mdash; present Arcade AI.</p>
+          <p>&copy; 2024 &mdash; present Arcade.</p>
```

Comment:
Is this legal?

Reply:
This is the only place we don't want to remove `AI`, because the legal company name is `Arcade AI, Inc`.

`&copy 2024 - present Arcade AI` is correct as far as I know.